BSP/Simulator/Src/BSP_CAN.c

```diff
@@ -28,6 +28,14 @@ void BSP_CAN_Init(callback_t rxEvent, callback_t txEnd, bool loopback) {
     Simulator_Log(LOG_INFO, "CAN Initialized\n");
 }
 
+/**
+ * @brief   Calls stm-level CAN_DeInit
+ * @return  None
+ */
+void BSP_CAN_DeInit() {
```

Comment:
You should also change the value of `CAN_Initialized` to false so more CAN messages aren't sent.

Reply:
fixed